tests/kademlia/utils.nim

```diff
@@ -65,7 +71,15 @@ proc containsNoData*(kad: KadDHT, key: Key): bool {.raises: [].} =
 template setupKadSwitch*(validator: untyped, selector: untyped): untyped =
   let switch = createSwitch()
   let kad = KadDHT.new(
-    switch, config = KadDHTConfig.new(validator, selector, timeout = chronos.seconds(1))
+    switch,
+    config = KadDHTConfig.new(
+      validator,
+      selector,
+      timeout = Timeout,
+      cleanupProvidersInterval = CleanupInterval,
+      providerExpirationInterval = ExpirationInterval,
+      republishProvidedKeysInterval = RepublishInterval,
```

Comment:
setting default values from const seem unnecessary here, unless there are more then one place where these const should be used (like before in sleeps).